tests/storage/merge_test.py

```diff
@@ -501,7 +501,11 @@ def failing_rebase(*args, **kw):
             assert subchain.top_vol.getLegality() == sc.ILLEGAL_VOL
 
     def test_reduce_chunked_internal(self):
-        with self.make_env(sd_type='block', format='cow', chain_len=4) as env:
+        with self.make_env(
+                sd_type='block',
+                format='cow',
+                chain_len=4,
+                size=3 * GiB) as env:
```

Comment:
Why did you add size here?

Reply:
With this implementation, capacities smaller than 2.5G (chunk size), allocated only one extent, and weren't reducing after merge. I changed it so that the size was bigger than the chunk size to get the reduce call in the test.

Reply:
This changes are best done in a separate patch, with commit message explaining why the change is needed. This makes the actual change smaller and safer and keeps the rule that every commit should have exactly one logical change.

Reply:
Finally removed this change.